benchexec/tablegenerator/react-table/src/tests/__snapshots__/Summary.test.js.snap

```diff
@@ -52612,7 +52612,7 @@ exports[`Render Summary for rows-with-scores.html 1`] = `
        >
         <div
          className="cell"
-         dangerouslySetInnerHTML={".128&#x2007;&#x2007;"}
```

Comment:
Seems like we are padding with whitespaces to match the maximum length of the values from the "values table" here.
I believe we should determine the width of the values using the displayed `sum` values of the corresponding column of the statistics table.

Reply:
Does this occur only for `rows-with-scores.html`? Then I suspect it is just because this manually created file is wrong.

Reply:
Hi Philipp, thanks for the response. 
The issue appears in `rows-with-scores.html` and `test.2015-03-03_1613.results.predicateAnalysis.custom-score.html`. 
Does the latter one also have a manually created part?

Reply:
`test.2015-03-03_1613.results.predicateAnalysis.custom-score.html` was not created manually, but there was a (different) bug leading to incorrect alignment, and I fixed it in a930f1f0. So please merge and then check again. For `rows-with-scores.html`, just fix the JSON in it manually.